modules/uncertainty/R/variance.R

```diff
@@ -47,7 +47,7 @@ get.gi.phii <- function(splinefuns, trait.samples, maxn = NULL){
   traits <- names(splinefuns)
   
   ## g_i(phi_i) the spline estimate of model output for value of trait i
-  gi.phii <- t(laply(traits, 
+  gi.phii <- plyr::laply(traits, 
```

Comment:
On this one, when you added the plyr:: it looks like you inadvertently deleted the transpose `t()`

Reply:
@Its-Maniaco Seconding Mike's comment. Here's a proposed fix -- if you agree it's the right one you can use the "commit suggestion" button below this comment to apply the fix immediately.

```suggestion
  gi.phii <- t(plyr::laply(traits, 
```

Reply:
Thank you for pointing out the mistake.